README.md

```diff
@@ -1,141 +1,60 @@
-# Australia API and Dashboard
+# Australian Bushfire API and Dashboard
 
 ## Background
 
-This project explores how wildfires have affected the country of Australia over time. Specifically, for this project, we gathered data captured by NASA satellites and various historical wildfire and climate data to help visualize what Australia not only has been through historically but also what Australia has gone through with the most recent (2019-2020) wildfire season. Go to our [website](https://australia-fire-api-dashboard.herokuapp.com/) to learn more about this project.
+This project explores how wildfires have affected Australia over time through data visualizations, including interactive time series charts, plots and a map. Visit our [website](https://australia-fire-api-dashboard.herokuapp.com/) to learn more.
 
-## Loading data into a database
+For this project, we collected open data from a number of sources, including data captured by NASA satellites and various wildfire and climate data. More information about our sources and the data itself can be found in the [Data page](./docs/data.md) in the `docs` folder.
+
+Following, we'll share a background of the project and explain how to recreate the webapp locally, using the code and data shared in this repo.
+
+### Technologies Used
+
+#### Infrastructure
+
+- [Flask](https://flask.palletsprojects.com/en/1.1.x/), a Python micro web framework used to run our API.
+- [MongoDB](https://www.mongodb.com/), a NoSQL for storing the data locally.
+- [MongoDB Atlas](https://www.mongodb.com/cloud/atlas), for storing the data in the cloud.
+- [Heroku](https://www.heroku.com/), for running our dashboard in the cloud.
```

Comment:
In general with technical documentation, you want to avoid using "we", "our", etc. I would just say "for running the application in the cloud."